lib/Dialect/FIRRTL/FIRRTLOps.cpp

```diff
@@ -2399,6 +2419,26 @@ LogicalResult RefDefineOp::verify() {
   return success();
 }
 
+static ParseResult parseConstDecay(OpAsmParser &parser, Type &destType,
```

Comment:
Thanks for this, much appreciate not having to always write/read a lot of extra types! (and less verbose IR is good for all)